code/modules/reagents/chemistry/recipes.dm

```diff
@@ -339,8 +397,10 @@
 		if((reagent.type in required_reagents) || (reagent.type in results))
 			reagents.add_reagent(reagent.type, reagent.volume, added_purity = reagent.purity, no_react = TRUE)
 			holder.remove_reagent(reagent.type, reagent.volume)
+	if(!force_range)
+		force_range = (sum_volume/5) + 3
```

Comment:
wasn't this just sum_volume/5 before?

Reply:
Yes, but in testing I realised that general volumes meant that the heater didn't smoke up the chemist. I'll make it vol/6 + 3